seth-shaw-unlv/controlled_access_terms
# Drupal 9

**GitHub Issue**: Resolves https://github.com/Islandora/documentation/issues/1679

# What does this Pull Request do?

Drupal 9 readiness.

# What's new?

* Changes deprecated methods for the new ones (courtesy of rector).
* Updates minor versions of required modules in composer to the Drupal-9 versions.
* Added `core_version_requirement: ^8 || ^9`.

# How should this be tested?

* Apply the PR
* `composer require drupal/geolocation:^3.2 drupal/token:^1.7`
* Run upgrade_status to ensure no known issues found: `composer require 'drupal/upgrade_status:^3.0' && drush en -y devel upgrade_status && drush us-a controlled_access_terms`

# Interested parties
@Islandora/8-x-committers, esp @kayakr
